pkg/io/storage/storage.go

```diff
@@ -0,0 +1,90 @@
+/*
+Package storage provides functions for storing and retrieving data.
+from json file
+*/
+package storage
+
+import (
+	"encoding/json"
+	"fmt"
+	"os"
+	"path/filepath"
+)
+
+type fileVault struct {
+	storage map[string][]byte
+	path    string
+	file    *os.File
+}
+
+func NewFileVault(path string) (*fileVault, error) {
+	var storage = make(map[string][]byte)
```

Comment:
It could be written in more concise way
```suggestion
	storage := make(map[string][]byte)
```